UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -52,17 +52,7 @@ private void Init()
 		layerMask = LayerMask.GetMask("Walls", "Door Closed");
 	}
 
-	private void OnEnable()
-	{
-		SceneManager.activeSceneChanged += OnSceneChange;
-	}
-
-	private void OnDisable()
-	{
-		SceneManager.activeSceneChanged -= OnSceneChange;
-	}
-
-	private void OnSceneChange(Scene oldScene, Scene newScene)
+	public void Clear()
```

Comment:
I presume this is called manually somewhere else?

Reply:
Yes. all "Clear" calls are later called in CleanupUtil. 
I just want all of this to be revieved and merged before I add calls one by one so we see could what gets messed up by what.